packages/strapi-plugin-users-permissions/config/layout.js

```diff
@@ -6,19 +6,5 @@ module.exports = {
       destroy: 'User.destroy',
       deleteall: 'User.destroyAll',
     },
-    attributes: {
```

Comment:
Is it normal to remove the default User layout?

Reply:
We are not going to use thoses classes anymore. We can define a new way to have default  layouts though. We didn't really talk about that. @Aurelsicoko @soupette might need to check that